src/zenml/integrations/mlflow/mlflow_utils.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from mlflow import (  # type: ignore[import]
+    ActiveRun,
+    get_experiment_by_name,
+    search_runs,
+    set_experiment,
+    set_tracking_uri,
+    start_run,
+)
+from mlflow.entities import Experiment  # type: ignore[import]
+
+from zenml.integrations.mlflow.experiment_trackers.mlflow_experiment_tracker import (
+    MLFlowExperimentTracker,
+)
+from zenml.logger import get_logger
+from zenml.repository import Repository
+
+logger = get_logger(__name__)
+
+MLFLOW_ENVIRONMENT_NAME = "mlflow"
```

Comment:
is this constant still needed ?

Reply:
Not sure this constant was ever needed tbh ;)

Reply:
Probably just copy paste from some other file